pom.xml

```diff
@@ -4,34 +4,18 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>beetle</artifactId>
-    <packaging>pom</packaging>
-
-    <version>0.2.0</version>
-
     <parent>
         <artifactId>insect</artifactId>
         <groupId>me.aurium</groupId>
-        <version>0.2.0</version>
+        <version>0.2.1</version>
     </parent>
 
-    <modules>
-        <module>core</module>
-        <module>core-default</module>
-        <module>spigot-impl</module>
-        <module>velo-impl</module>
-        <module>discord-impl</module>
-        <module>spigot-examples</module>
+    <artifactId>beetle</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
 
-    </modules>
+    <packaging>pom</packaging>
 
     <properties>
-
-        <beetle.version>0.2.0</beetle.version>
-
-        <core.version>0.3.3</core.version>
-        <default.version>0.3.3</default.version>
-
         <spigot.version>1.16.5-R0.1-SNAPSHOT</spigot.version>
         <logger.version>1.7.25</logger.version>
 
```

Comment:
Junit 4.10 for new code?

Reply:
i love junit 4 and i'm more used to it - it was second nature for me to add it in here. That being said, i should probably get used to junit 5 since i'd like to keep everything up to date.